boot/bootutil/include/bootutil/nrf_partitions.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2025 JUUL Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef H_NSIB_PARTITIONS_
+#define H_NSIB_PARTITIONS_
+
+#if defined(CONFIG_SECURE_BOOT_STATIC_DTS)
```

Comment:
do not add a pointless symbol for this, check if partition manager is not set

Reply:
Fixed now